frontend/src/store/index.js

```diff
@@ -23,8 +20,26 @@ var flashInt
 Vue.use(Vuex)
 
 export const store = new Vuex.Store({
-
+  
```

Comment:
What happened on this file? Can you revert this and just add in the actual code changes? Some white-space fuckery happened here.

Reply:
will do.